src/std_extensions/arithmetic/int_ops.rs

```diff
@@ -11,6 +11,7 @@ use crate::ops::OpName;
 use crate::type_row;
 use crate::types::{FunctionType, PolyFuncType};
 use crate::utils::collect_array;
+
```

Comment:
Out of curiosity: are you following some style guide regarding newlines here? Would like to know for my own coding if there is a recommended style.